komand/server.py

```diff
@@ -33,6 +46,16 @@ def __init__(self, plugin, port=10001, workers=1, threads=4, debug=False):
         self.plugin = plugin
         self.debug = debug
         self.app = self.create_flask_app()
+        self.workers = workers
+        self.threads = threads
+
+        # Create an APISpec
+        self.spec = APISpec(
+            title="Runtime API",
+            version="1.0.0",
+            openapi_version="3.0.2",
```

Comment:
Rapid7 standardizes on version `2.0` at the moment.
```suggestion
            openapi_version="2.0",
```